From 696a1fefa9f4e320083d6a972450ee36ee35f3b6 Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Tue, 19 Dec 2023 14:05:27 +0100
Subject: [PATCH 1067/1141] dt-bindings: media: st,stm32mp25-venc: add
 'power-domains' property

STM32 VENC may be in a power domain which is the case for the STM32MP25
based boards. Allow a single 'power-domains' entry for STM32 VENC.

Fix dtbs_check warning:
 Unevaluated properties are not allowed ('power-domains' was unexpected)

Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Change-Id: I1a113b4a200ffeae41d55698780f9e325eca047c
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/347871
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Philippe CORNU <philippe.cornu@foss.st.com>
Domain-Review: Philippe CORNU <philippe.cornu@foss.st.com>
---
 Documentation/devicetree/bindings/media/st,stm32mp25-venc.yaml | 3 +++
 1 file changed, 3 insertions(+)

--- a/Documentation/devicetree/bindings/media/st,stm32mp25-venc.yaml
+++ b/Documentation/devicetree/bindings/media/st,stm32mp25-venc.yaml
@@ -36,6 +36,9 @@ properties:
   clock-names:
     maxItems: 1
 
+  power-domains:
+    maxItems: 1
+
 required:
   - compatible
   - reg
